sys/auto_init/auto_init.c

```diff
@@ -280,6 +280,11 @@ AUTO_INIT(auto_init_atca,
           AUTO_INIT_PRIO_MOD_CRYPTOAUTHLIB);
 #endif
 #endif
+#if IS_USED(MODULE_DRIVER_CRYPTOCELL)
+extern void cryptocell_setup(void);
+AUTO_INIT(cryptocell_setup,
```

Comment:
Shouldn't this be part of the `nrf` init?

Reply:
I added this here, because it initializes the cryptocell library, which is a third party software package and not really a part of the CPU. Also it's not automatically used, but only in combination with the package.
Should I maybe change the naming to something like `cryptocell_library_setup`?

Reply:
done